specs/agents/tracing-spans-service-target.md

```diff
@@ -105,35 +105,37 @@ This specification assumes that values for `span.type` and `span.subtype` fit th
 - `span.context.service.target.name` depends on the span context attributes
 
 On agents, the following algorithm should be used to infer the values for `span.context.service.target.*` fields.
+
```

Comment:
REVIEW NOTE: This section slightly tweaks the pseudo-code to infer `context.service.target.*` from other span fields to avoid a couple gotchas that I hit:
1. The `if (!service_target.type)` check in JavaScript would accidentally override `service_target.type` if it had explicitly been set to the empty string.
2. The `if (span.context.db.instance)` branch was changed to `if (span.context.db)` to use the DB logic for DB spans even if the span didn't happen to have a "db.instance" set -- which is less surprising and matches the old pseudo-code for inferring destination.service.resource.